docs/CONTRIBUTING.md

```diff
@@ -152,7 +152,7 @@ when the Operator will be installed, and it contains:
 
 ## Customize Builder Image
 
-At the startup a [Dockerfile](../config/manager/sonataflow_builder_dockerfile.yaml) is placed in a configmap. This
+At the startup a [Dockerfile](../config/manager/app/sonataflow_builder_dockerfile.yaml) is placed in a configmap. This
```

Comment:
I'd rather not do this. If you want, we can discuss if it would be better to create another branch/repo for productization.

Doing so will break operator-sdk/kubebuilder scaffolding.

Reply:
This is not breaking anything if managed like we are doing.

Having another branch only for the productization will add a lot of effort due to the branch alignments and will prevent us to have smart CI checks that could let us know if the code is everytime ready to be productized.

Anyway we can discuss about the 2 approaches.